config/level.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"strings"
+)
+
+type Level int
+
+const (
+	DebugLevel Level = iota
+	InfoLevel
+	WarnLevel
+	ErrorLevel
+	PanicLevel
+	UnknownLevel // unknown should be higher than everything
+)
+
+func ParseLevel(s string) Level {
+	sanitizedLevel := strings.TrimSpace(strings.ToLower(s))
+	switch sanitizedLevel {
+	case "debug":
+		return DebugLevel
+	case "info":
+		return InfoLevel
+	case "warn", "warning":
```

Comment:
I was maybe being too clever, trying to be permissive about what I thought would be a likely error -- but yeah, we shouldn't do that especially since we're NOT being permissive about other things. Will fix.